EncounterMe/EncounterMe/Views/FavouritesPage.xaml.cs

```diff
@@ -31,6 +31,10 @@ public FavouritesPage()
 
                 User = App.s_userDb.GetUserByID((int)App.s_userDb.CurrentUserId);
             }
+            else
+            {
+                DisplayAlert("Favorites", "Currently you have no favorite objects", "Ok");
```

Comment:
Šitas nereikalingas, nes jei pažiūrėtum funkciją OnAppearing, tai ten, jei user'is neturi favourite objects, tai pridedamas label tam tikras, kuriam ir rašo, kad dar neturi favourite objects. O ir if tikrinimas nelabai atitinka else esančios alert žinutės.
Čia gal būtų protingiau  tada rodyt alert, kuris sakytų, jog negalėjom išvis gauti user'io duomenų, o ne jo favourite objektų ir tada tą patį alert pridėt prie to paties tikrinimo OnAppearing funkcijoj.

Reply:
Ok istrinsiu

Reply:
Tai gal tu ten jei manai, kad reikia, galesi padaryt. Geriau vis tiek supranti sita klase, nei as

Reply:
Netrink, o tik pakeisk alert žinutę

Reply:
Tai čia šitas tavo prirašymas yra geras, tiesiog reikia pakeist gal alert žinutę ir viskas tiks

Reply:
ok